src/main/java/com/exasol/dbbuilder/objectwriter/DatabaseObjectWriter.java

```diff
@@ -107,5 +109,12 @@
      * @param parameters script parameters
      * @return table
      */
-    public List<List<Object>> executeQuery(Script script, Object... parameters);
+    public List<List<Object>> executeQuery(AbstractDatabaseObject script, Object... parameters);
+
+    /**
+     * Execute the contents of a SQL script file.
+     *
+     * @param sqlFile path to the script file
+     */
+    public void executeSql(Path sqlFile);
```

Comment:
`executeSql` sounds suspicious. Maybe `executeScriptFile` will be clearer?

Reply:
The reason is that I wanted to avoid confusion between the term "script" and a file containing SQL statements. We don't know what is in that SQL file. Could be schema and table creation plus inserts for all we know.

I actually planned to overload `executeSql` so that you can either feed it with a file or a string.
But since that name confused you, how about `executeSqlFile`?

Reply:
Yes, I think `executeSqlFile` is better